_po/ja/how-to/laravel.po

```diff
@@ -68,30 +86,43 @@ msgid ""
 " web page quickly.\n"
 "To install Laravel Breeze, just run the following commands:"
 msgstr ""
+"次にLaravel Breezeをインストールします。ログインやユーザ登録、パスワードリセットや"
+"Email確認機能、パスワード確認機能を兼ね備えたシンプルな認証パッケージです。"
+"TailwindCSSによりデザインされたBladeテンプレートが準備されているため"
+"WEBページのデザインを素早く作ることができます。"
+"次のコマンドを実行してLaravel Breezeをインストールします:"
 
 msgid ""
 "```shell\n"
 "composer require laravel/breeze --dev\n"
 "php artisan breeze:install blade\n"
 "```"
 msgstr ""
+"```shell\n"
+"composer require laravel/breeze --dev\n"
+"php artisan breeze:install blade\n"
+"```"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.